src/test/javascript/spec/component/settings/ide/ide-settings.component.spec.ts

```diff
@@ -20,8 +21,7 @@ describe('IdeSettingsComponent', () => {
 
     beforeEach(async () => {
         await TestBed.configureTestingModule({
-            imports: [],
-            declarations: [IdeSettingsComponent],
+            imports: [TranslateTestingModule],
```

Comment:
Just very much a nit-pick but can is it possible to mock this module in the declarations using `MockModule(`TranslateTestingModule`)?